recipes-kernel/linux/linux-rockchip_5.10/0022-drivers-rtc-add-support-for-max31343-rtc.patch

```diff
@@ -0,0 +1,576 @@
+From 4e62c4d86c9c549bbd0afad183a6efc0a8094844 Mon Sep 17 00:00:00 2001
+From: Christian Biermann <bie@iesy.com>
+Date: Fri, 27 Oct 2023 14:43:48 +0200
+Subject: [PATCH 1/1] drivers: rtc: add support for MAX31343 rtc
+
+---
+ .../boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts    |   9 +
+ drivers/rtc/Kconfig                           |  10 +
+ drivers/rtc/Makefile                          |   1 +
+ drivers/rtc/rtc-max31343.c                    | 494 ++++++++++++++++++
+ 4 files changed, 514 insertions(+)
+ create mode 100644 drivers/rtc/rtc-max31343.c
+
+diff --git a/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts b/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts
+index d1ed19e2f84e..45515d06117e 100644
+--- a/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts
++++ b/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts
+@@ -625,6 +625,13 @@ rtc@51 {
+ 		reg = <0x51>;
+ 	};
+ 
++	/* RTC on CM006 REV3.00 */
++	rtc@68 {
++		status = "okay";
++		compatible = "adi,max31343";
```

Comment:
compatible string does not match with the string given in the driver, "max,max31343"

Reply:
Driver seems not to use the compatible string.